<launch>
    <arg name="save" default="false"/>
    <node pkg="rosbag" type="play" name="rosbag" required="true" args="$(find pyx4_avoidance)/src/nn/bags/$(arg save).bag"/>
    <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
      <remap from="image" to="/pyx4_avoidance_node/optic_flow_draw"/>
    </node>
  </launch>